ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml

```diff
@@ -451,7 +451,7 @@ tests:
         remote command execution over websockets \| Pods should support retrieving
         logs from the container over websockets \| NetworkPolicy between server and
         client should deny ingress access to updated pod \| Image append should create
-        images by appending them \| In-tree Volumes
+        images by appending them \| In-tree Volumes \| Only known images used by tests
```

Comment:
We just added configs for 4.10 (ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml), can you add it there as well?